Letterbook.Core/Exceptions/ErrorCodes.cs

```diff
@@ -1,12 +1,21 @@
 ﻿namespace Letterbook.Core.Exceptions;
 
+/// <summary>
+/// </summary>
 [Flags]
 public enum ErrorCodes
```

Comment:
Do we really want numeric error codes? They can make exceptions and error logs harder to read.

Reply:
I hope #114 covers that. I'd love to discuss error handling as it's own topic.